internal/executor/executor.go

```diff
@@ -170,26 +172,16 @@ func (executor *Executor) RunBuild(ctx context.Context) {
 
 		log.Printf("%s finished!", command.Name)
 
-		_ = retry.Do(
-			func() error {
-				_, err := client.CirrusClient.ReportSingleCommand(ctx, &api.ReportSingleCommandRequest{
```

Comment:
We still need to report a single command to update UI. We can stop retrying though...

Reply:
Reverted in https://github.com/cirruslabs/cirrus-ci-agent/pull/143/commits/ba021cdb3a6115c0c838c57f35cf62f7197a16d7, but without retries.